test/e2e/operator_test.go

```diff
@@ -1111,6 +1178,25 @@ func TestAWSLoadBalancerControllerWithExternalTypeNLBAndNonStandardPort(t *testi
 	}
 }
 
+// TestAWSLoadBalancerControllerOpenAPIValidation tests validations added to AWSLoadBalancerController CRD.
+func TestAWSLoadBalancerControllerOpenAPIValidation(t *testing.T) {
```

Comment:
I think this test is a good addition, but future looking, are you familiar with the API integration test suite? https://github.com/openshift/api/tree/master/tests#readme. You can create API tests like this in YAML files. Not sure how easy it would be to port that over here.

Reply:
Interesting. That may be useful to test the openAPI validations indeed. The setup may be a little heavy for this concrete PR but if the API will grow in complexity we can think of it.